ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudConfig.java

```diff
@@ -52,7 +52,7 @@
   public final String cosmosKey;
 
   public AzureCloudConfig(VerifiableProperties verifiableProperties) {
-    storageConnectionString = verifiableProperties.getString(STORAGE_CONNECTION_STRING);
+    azureStorageConnectionString = verifiableProperties.getString(AZURE_STORAGE_CONNECTION_STRING);
```

Comment:
quick question: why these configs have no default value?

Reply:
There is no default, it's infra-dependent.